curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66cbe2319d3845545a293a0b.md

```diff
@@ -16,7 +16,7 @@ Add a `console.log` and call the `getShoppingListMsg` function inside of the `co
 You should call the `getShoppingListMsg` function inside of the `console.log`.
```

Comment:
the hint needs to be updated to mention the argument, or it needs a separate test that check specifically for the argument